src/lib/ops/organizationOps.js

```diff
@@ -200,7 +202,9 @@ var organizationOps = {
 
     const tasks = [
       (job, next) => {
-        const { id, name, description, user } = params
+        const id = params.id || uuid()
```

Comment:
Did we decided to switch to an optional organization id? I have miss that :/

I would keep the organization id `required` and let the caller specify it.